docs/src/app/components/Dialog/README.md

```diff
@@ -0,0 +1,6 @@
+## Dialog
+
+Dialog can only be a controlled component.
```

Comment:
Could you add a link to https://www.google.com/design/spec/components/dialogs.html?